src/test/java/org/fundacionjala/coding/abel/SrotTheInnerCtonnetInDsnnieedcgOredrTest.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.abel;
+
+import static org.junit.Assert.assertEquals;
+
+
+import org.junit.Test;
+
+/**
+ *
+ */
+public class SrotTheInnerCtonnetInDsnnieedcgOredrTest {
+    /**
+     *
+     */
+
+
+    @Test
+    public void testSortTheInnerContent() {
+        SrotTheInnerCtonnetInDsnnieedcgOredr instance = new SrotTheInnerCtonnetInDsnnieedcgOredr();
```

Comment:
You are repeating this line of code in all your tests.
Please move it to an @Before method.